res/values-ro/strings.xml

```diff
@@ -128,7 +128,7 @@
     <string name="Friend_Request_Sent">"Cerere de prietenie trimisă"</string>
     <string name="ERROR">"EROARE"</string>
     <string name="CONTINUE">"CONTINUĂ"</string>
-    <string name="Global_" formatted="false">"Global%"</string>
+    <string name="Global_" formatted="false">"Global formatat fals"</string>
```

Comment:
Wrong change here. You shouldn't translate and provide the `formatted="false"`.

Reply:
Alright

Reply:
So is not right, why, that said there Without%

Reply:
There up at winner is Câstigator not how they translate there, can i get my tag translator or i need to try again and to see better there if is wrong or no something

Reply:
You just need to correct the changes

Reply:
Alright, i changed right now